aaq/tasks.py

```diff
@@ -0,0 +1,35 @@
+import requests
+from celery.exceptions import SoftTimeLimitExceeded
+from django.conf import settings
+from requests.exceptions import RequestException
+from temba_client.exceptions import TembaHttpError
+
+from ndoh_hub.celery import app
+
+
+@app.task(
+    autoretry_for=(RequestException, SoftTimeLimitExceeded, TembaHttpError),
+    retry_backoff=True,
+    max_retries=15,
+    acks_late=True,
+    soft_time_limit=10,
+    time_limit=15,
+)
+def send_feedback_task(secret_key, inbound_id, feedback_type, **kwargs):
+    data = {
+        "feedback_secret_key": secret_key,
+        "inbound_id": inbound_id,
+        "feedback": {"feedback_type": feedback_type},
+    }
+    if "faq_id" in kwargs:
+        data["feedback"]["faq_id"] = kwargs["faq_id"]
+    if "page" in kwargs:
+        data["feedback"]["page_number"] = kwargs["page"]
+
+    url = f"{settings.AAQ_CORE_API_URL}/inbound/feedback"
```

Comment:
You can use `urllib.parse.urljoin` here instead, that way it will still work if `AAQ_CORE_API_URL` is configured with trialing slash or not.

Reply:
Done